tests/storage-spec.js

```diff
@@ -155,7 +163,7 @@ describe('languageStorage factory', function () {
                     label: 'Polski',
                 },
                 {
-                    id: 'en_us',
+                    id: 'en',
```

Comment:
I think this can be added as an improvement when we actually need it, since I think it's very unlikely considering our use cases